kotlin-checks-test-sources/src/main/kotlin/checks/VarShouldBeValCheckSample.kt

```diff
@@ -0,0 +1,114 @@
+package checks
+
+import kotlin.math.max
+
+class VarShouldBeValCheckSample {
```

Comment:
Let's add a `lateinit var` sample and not raise anything there

Reply:
:+1: